subprojects/degrees-and-programs/source/cascade/formats/Degrees-programs/PrimaryContent.vm

```diff
@@ -272,13 +269,15 @@
 
 										#foreach ( $motivation in $motivations )
 
+											#set ( $rand = $_MathTool.floor( $_MathTool.mul( $_MathTool.getRandom(), 10000 ) ) )
```

Comment:
Thanks for explanation. Can you add that as a comment to code? First paragraph will be sufficient:

    // Random number helps create a unique ID for each motivation's corresponding label and input field. 
    // Same with interests below.